### PR TITLE
FixNpcNoclip

### PR DESCRIPTION
**Pull request by louis1706**
## Description
**Describe the changes** 

Fix NpcNoclip that can't be moved

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Other
- [ ] Still requires more testing

### CODE REVIEW

File: EXILED/Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -506,7 +506,12 @@ public Player Cuffer
         public Vector3 Position
         {
             get => Transform.position;
-            set => ReferenceHub.TryOverridePosition(value, Vector3.zero);
+            set
+            {
+                ReferenceHub.TryOverridePosition(value, Vector3.zero);
+                if (Role is FpcRole fpcRole)
+                    fpcRole.RelativePosition = new(value);
```
Comment (IRacle1) [inline EXILED/Exiled.API/Features/Player.cs:513]: Will be better to exec that code only for npc, bcz i dont like the idea of forcing that for all players

**Review by IRacle1 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.API/Features/Player.cs (hunk 1 above)
Comment (louis1706) [reply EXILED/Exiled.API/Features/Player.cs:513]: > Will be better to exec that code only for npc, bcz i dont like the idea of forcing that for all players

oh wait i was thinking making it NPC class oups

**Comment by IRacle1** [comment]
Better will be to virtualize it 🙂

**Comment by louis1706** [comment]
> Better will be to virtualize it 🙂 Shouldnt be breaking

uhm you sure ?

**Comment by louis1706** [comment]
> > Better will be to virtualize it 🙂 Shouldnt be breaking
> 
> uhm you sure ?

I can't modify it anymore you will need to commit on my pr

**Comment by IRacle1** [comment]
> > > Better will be to virtualize it 🙂 Shouldnt be breaking
> > 
> > 
> > uhm you sure ?
> 
> I can't modify it anymore you will need to commit on my pr

oh i didnt saw it, ill finish that tmrw

**Review by Misfiy — APPROVED** [review]

**Pull request merged**